example/styles.css

```diff
@@ -65,3 +65,85 @@ a:hover, a:active {
 #map-container *::after {
   box-sizing: content-box;
 }
+
+.tabs-body {
```

Comment:
Most of this is stolen from core-ui tabs but I had to tweak it a lot to get it to override the bootstrap styles properly.